moveit_core/robot_model/CMakeLists.txt

```diff
@@ -12,20 +13,25 @@ add_library(${MOVEIT_LIB_NAME}
   src/revolute_joint_model.cpp
   src/robot_model.cpp
   )
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME} moveit_profiler moveit_exceptions moveit_kinematics_base ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+  urdf
+  urdfdom_headers
+  visualization_msgs
+  moveit_profiler
+  moveit_exceptions
+  moveit_kinematics_base)
 
-if(CATKIN_ENABLE_TESTING)
-  find_package(moveit_resources REQUIRED)
-  include_directories(${moveit_resources_INCLUDE_DIRS})
-
-  catkin_add_gtest(test_robot_model test/test.cpp)
-  target_link_libraries(test_robot_model moveit_test_utils ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${MOVEIT_LIB_NAME})
-endif()
+# if(BUILD_TESTING)
+#   find_package(ament_cmake_gtest REQUIRED)
+#   find_package(moveit_resources REQUIRED)
+#   include_directories(${moveit_resources_INCLUDE_DIRS})
+#
+#   ament_add_gtest(test_robot_model test/test.cpp)
+#   target_link_libraries(test_robot_model moveit_test_utils ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${MOVEIT_LIB_NAME})
+# endif()
```

Comment:
let's not disable the few tests we have, that only decreases quality of software

Reply:
Resolving this, discussion about issues with unit tests follows at https://github.com/AcutronicRobotics/moveit2/issues/13